To: vim_dev@googlegroups.com
Subject: Patch 7.3.435
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.435
Problem:    Compiler warning for unused variable.
Solution:   Move the variable inside #ifdef.
Files:      src/ex_cmds2.c


*** ../vim-7.3.434/src/ex_cmds2.c	2012-02-05 23:10:25.000000000 +0100
--- src/ex_cmds2.c	2012-02-06 06:16:28.000000000 +0100
***************
*** 3400,3406 ****
  {
      struct source_cookie *sp = (struct source_cookie *)cookie;
      char_u		*line;
!     char_u		*p, *s;
  
  #ifdef FEAT_EVAL
      /* If breakpoints have been added/deleted need to check for it. */
--- 3400,3406 ----
  {
      struct source_cookie *sp = (struct source_cookie *)cookie;
      char_u		*line;
!     char_u		*p;
  
  #ifdef FEAT_EVAL
      /* If breakpoints have been added/deleted need to check for it. */
***************
*** 3471,3476 ****
--- 3471,3478 ----
  #ifdef FEAT_MBYTE
      if (line != NULL && sp->conv.vc_type != CONV_NONE)
      {
+ 	char_u	*s;
+ 
  	/* Convert the encoding of the script line. */
  	s = string_convert(&sp->conv, line, NULL);
  	if (s != NULL)
*** ../vim-7.3.434/src/version.c	2012-02-06 00:13:16.000000000 +0100
--- src/version.c	2012-02-11 20:38:49.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     435,
  /**/

-- 
Light travels faster than sound.  This is why some people
appear bright until you hear them speak

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
